plugins/notifications-backend-module-slack/src/lib/constants.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @public
+ */
+export const ANNOTATION_SLACK_NOTIFICATIONS = 'slack.com/notifications';
```

Comment:
This is redundant, as where to send a notification is configured in the workflow.
Instead of this link, the code has to point to the `channelWebhookUrl`